### PR TITLE
Hotfix/376 qr pdf display incorrect

### PR DESCRIPTION
**Pull request by CalderBriar**
## Changes made

- Fix issue of QR and PDF displaying instead of a prompt for user action for adding PDF or QR
- Small code refactor
- Move styling to proper place

## Screenshots

(prefer animated gif)

## Test path (if necessary)

1. Go to this screen.
2. Try using QR or PDF

## Checklist

- [ ] rebased feature branch with develop branch
- [x] tested locally
  - [x] in emulator
  - [ ] on a real device
- [ ] added new dependencies (what deps?)
- [ ] used ESLint (linted) and Prettier (formatted)
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: src/domains/transport/components/TransportItem/TransportItem.js [hunk 1]
```diff
@@ -32,10 +32,11 @@ const TransportItem = (props) => {
 
   var source = {uri: pdfUri};
```
Comment (weronikaolejniczak) [inline src/domains/transport/components/TransportItem/TransportItem.js:32]: Change var to const if it's not reassigned or let if it is reassigned.
var shouldn't be used.

File: src/domains/transport/components/TransportItem/TransportItem.js [hunk 2]
```diff
@@ -192,7 +193,7 @@ const TransportItem = (props) => {
 
         <TouchableOpacity
           onPress={async () => {
```
Comment (weronikaolejniczak) [inline src/domains/transport/components/TransportItem/TransportItem.js:193]: This doesn't need async keyword.

File: src/domains/transport/components/TransportItem/TransportItemStyle.js [hunk 3]
```diff
@@ -104,7 +102,7 @@ export const styles = StyleSheet.create({
     alignItems: 'center',
     //alignContent: 'center',
```
Comment (weronikaolejniczak) [inline src/domains/transport/components/TransportItem/TransportItemStyle.js]: These commented style instructions should be removed.

File: src/domains/transport/components/TransportItem/TransportItem.js [hunk 4]
```diff
@@ -145,7 +145,7 @@ const TransportItem = (props) => {
             source={source}
             onLoadComplete={(numberOfPages, filePath) => {
               //console.log(`number of pages: ${numberOfPages}`);
```
Comment (weronikaolejniczak) [inline src/domains/transport/components/TransportItem/TransportItem.js:146]: Delete commented console.logs as well. Also, these methods don't do anything? If so, I'd remove them and make for example a wiki about them, to explain that they can be there, what they do etc.

**Review by mkowalczyk97 — APPROVED** [review]

**Review by weronikaolejniczak — CHANGES_REQUESTED** [review]
Some small changes are necessary. Otherwise, it looks good. 👍🏻 
I really like that you removed inline styles and put them in the styling file.

**Review by weronikaolejniczak — APPROVED** [review]

**Pull request merged**